proto/encore/parser/meta/v1/meta.proto

```diff
@@ -8,56 +8,56 @@ import "encore/parser/schema/v1/schema.proto";
 
 // Data is the metadata associated with an app version.
 message Data {
-  string module_path = 1; // app module path
-	string app_version = 2;
-  repeated schema.v1.Decl decls = 3;
-  repeated Package pkgs = 4;
-  repeated Service svcs = 5;
-  AuthHandler auth_handler = 6;
+  string                  module_path  = 1; // app module path
+	string                  app_version  = 2;
```

Comment:
The leading indentation here is screwed up (my bad; it's part of the previous commit as well), but would be good to fix